saleor/checkout/views/summary.py

```diff
@@ -46,6 +46,11 @@ def handle_order_placement(request, checkout):
     return redirect_url
 
 
+def get_contract_acceptance_form(data):
+    contract_form = ContractAcceptanceForm(data or None)
```

Comment:
As data is required argument here, I think `or None` can be removed.

Reply:
If you remove `or None` then the form will be invalid by default.